cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,130 @@
+        <li>Regularly test security systems and processes.</li>
+      </ol>
+  </td>
+</tr>
+<tr>
+  <td>Maintain an Information Security Policy</td>
+  <td><ol start="12">
+        <li>Maintain a policy that addresses information security for all personnel.</li>
+  </td>
+</tr>
+</tbody>
+</tgroup>
+</table>
+
+{{ site.data.products.dedicated }} is certified by a PCI Qualified Security Assessor (QSA) to be compliant with [PCI DSS 3.2.1](https://listings.pcisecuritystandards.org/documents/SAQ_D_v3_Merchant.pdf), as long as you take the actions outlined in [Responsibilities of the customer](#responsibilities-of-the-customer).
+
+## Responsibilities of Cockroach Labs
+
+Cockroach Labs takes actions to ensure that our operating procedures and the deployment environment for {{ site.data.products.dedicated }} clusters meet or exceed the requirements of PCI DSS 3.2.1. Some of these actions include:
+
+- Enforcing comprehensive security policies and standards.
+- Providing periodic security training for all Cockroach Labs employees.
+- Hardening our operating environments and networks according to industry standards and recommended practices, to ensure that they are secure and resilient against vulnerabilities and attacks.
+- Encrypting cluster data and metadata at rest and in transit.
+- Regularly scanning our environment using vendors designated by PCI as [Approved Scanning Vendors (ASVs)](https://www.pcidssguide.com/what-is-a-pci-approved-scanning-vendor-asv/) to ensure our continued compliance with PCI DSS 3.2.1, and correcting issues as quickly as possible.
+- Regularly scanning our environment and software for known security vulnerabilities and applying updates and security patches in a timely manner.
+- Implementing [data loss prevention (DLP)](https://pcidss.com/listing-category/data-loss-protection-dlp).
+- Logging cluster actions and events, redacting sensitive information in audit logs, and retaining audit logs according to the [logging requirements of PCI DSS](https://listings.pcisecuritystandards.org/documents/Effective-Daily-Log-Monitoring-Guidance.pdf).
+
+A comprehensive list of all actions that Cockroach Labs takes to ensure compliance with PCI DSS 3.2.1 is beyond the scope of this document. For more information, contact your Cockroach Labs account team.
+
+Cockroach Labs automatically and regularly applies security and stability updates to {{ site.data.products.dedicated }} clusters. These updates include, but are not limited to, the cluster's CockroachDB runtimes and APIs, the operating systems of cluster nodes, and management utilities. You are notified about upcoming cluster maintenance before it happens, when it starts, and when it completes. Refer to [Monitoring and Alerting](/docs/stable/monitoring-and-alerting.html). You can also read the release notes for [CockroachDB](/docs/releases/index.html) and for [{{ site.data.products.db }}](/docs/releases/cloud.html).
```

Comment:
I would say remove this section. Applying security patches is already indicated in the list above, and having a specific point just on that would attract unnecessary attention. Especially when some of our customers are annoyed by maintenance related restarts - which will be resolved through maintenance windows coming up soon.